examples/gogo/command/src/main/java/org/osgi/enroute/examples/gogo/command/MyCommand.java

```diff
@@ -0,0 +1,19 @@
+
+package org.osgi.enroute.examples.gogo.command;
+
+import org.osgi.enroute.examples.gogo.command.upstream.annotations.RequireCommand;
+import org.osgi.enroute.examples.gogo.command.upstream.propertytypes.Command;
+import org.osgi.service.component.annotations.Component;
+
+@Component(service = { MyCommand.class } /* You need to provide a class, without it will not be picked up. */)
+@Command(scope = "mycmd", function = { "foo", "bar" })
+@RequireCommand
```

Comment:
I tried to use an annotation for the requirement(s).

Reply:
Could the `@RequireCommand`  annotation be applied as a meta-annotation on the `@Command`? That would be more elegant and harder for people to forget to do.